moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -740,6 +749,51 @@ bool parse(const YAML::Node& node, const std::string& key, T& storage, const T&
   return valid;
 }
 
+bool MoveItConfigData::inputOMPLYAML(const std::string& file_path)
+{
+  // Load file
+  std::ifstream input_stream(file_path.c_str());
+  if (!input_stream.good())
+  {
+    ROS_ERROR_STREAM("Unable to open file for reading " << file_path);
+    return false;
+  }
+
+  // Begin parsing
+  try
+  {
+    YAML::Node doc = YAML::Load(input_stream);
+
+    // Loop through all groups
+    for (YAML::const_iterator group_it = doc.begin(); group_it != doc.end(); ++group_it)
+    {
+      // get group name
+      const std::string group_name = group_it->first.as<std::string>();
+
+      // compare group name found to list of groups in group_meta_data_
+      std::map<std::string, GroupMetaData>::iterator group_meta_it;
+      group_meta_it = group_meta_data_.find(group_name);
+      if (group_meta_it != group_meta_data_.end())
+      {
+        std::string planner;
+        parse(group_it->second, "default_planner_config", planner);
+        int pos = planner.find_last_not_of("kConfigDefault");
```

Comment:
I suggest not to cut this `kConfigDefault` suffix. Usually it's this name, but people may change it.